src/main/java/yablunovskyi/bookstore/controller/BookController.java

```diff
@@ -27,31 +31,41 @@ public class BookController {
     private final BookService bookService;
     
     @ResponseStatus(HttpStatus.OK)
+    @Operation(summary = "Get all books",
+            description = "Get a list of all available books from the datase")
     @GetMapping
-    public List<BookDto> findAll() {
-        return bookService.findAll();
+    public List<BookDto> findAll(Pageable pageable) {
```

Comment:
you may also use `@PageableDefault` in case client did not provide pagination params